opencsp/app/sofast/lib/AbstractMeasurementSofast.py

```diff
@@ -0,0 +1,98 @@
+"""Measurement class for SofastFringe
+"""
+
+from abc import ABC, abstractmethod
+import datetime as dt
+
+import numpy as np
+
+from opencsp.app.sofast.lib.ImageCalibrationAbstract import ImageCalibrationAbstract
+import opencsp.app.sofast.lib.DistanceOpticScreen as sod
+from opencsp.common.lib.geometry.Vxyz import Vxyz
+import opencsp.common.lib.tool.hdf5_tools as h5
```

Comment:
Could you remove unused imports:

```suggestion
from abc import ABC
import datetime as dt

import opencsp.app.sofast.lib.DistanceOpticScreen as sod
import opencsp.common.lib.tool.hdf5_tools as h5
```

Reply:
Done! Thanks for the suggested change!